channels/api_test.py

```diff
@@ -576,6 +577,73 @@ def test_api_constructor(mocker, settings, verify_ssl):
     assert config.refresh_token == refresh_token
 
 
+def test_api_constructor_error(mocker):
+    """
+    If a non-401 response is received during Api() initialization, we should delete the refresh
```

Comment:
I think this comment is supposed to read `we should not delete`

Reply:
haha oops